picard/tagger.py

```diff
@@ -234,6 +245,18 @@ def __init__(self, picard_args, localedir, autoupdate, pipe_handler=None):
         # Default thread pool
         self.thread_pool = ThreadPoolExecutor()
 
+        self.commands = {
+            "CLUSTER": self.handle_command_cluster,
+            "FINGERPRINT": self.handle_command_fingerprint,
+            "LOOKUP": self.handle_command_lookup,
+            "QUIT": self.handle_command_quit,
+            "REMOVE_SAVED": self.handle_command_remove_saved,
+            "SAVE_COMPLETE": self.handle_command_save_complete,
+            "SCAN": self.handle_command_scan,
+            "SHOW": self.handle_command_show,
+            "SUBMIT_FINGERPRINTS": self.handle_command_submit_fingerprints,
+        }
+
```

Comment:
Perhaps it would be easier to move this near `handle_command_*()` methods:

Here call:
```python
    self._init_remote_commands()
```

Near  `handle_command_*()` methods:

```python
    def _init_remote_commands(self):
        self.commands = {
            "CLUSTER": self.handle_command_cluster,
            "FINGERPRINT": self.handle_command_fingerprint,
            "LOOKUP": self.handle_command_lookup,
            "QUIT": self.handle_command_quit,
            "REMOVE_SAVED": self.handle_command_remove_saved,
            "SAVE_COMPLETE": self.handle_command_save_complete,
            "SCAN": self.handle_command_scan,
            "SHOW": self.handle_command_show,
            "SUBMIT_FINGERPRINTS": self.handle_command_submit_fingerprints,
        }
```